### DISCUSSION TITLE
Bring liblrzip up to date

### DISCUSSION THREAD

**Discussion by pete4abw** [pete4abw/lrzip-next#14] [Ideas]
With the enhancements in version 0.7x of **lrzip**, these options are omitted from the lrzip library, **liblrzip**. In particular, pre-compression filtering, and the ability to override default dictionary sizes. The following options need to be accounted for:

```
--x86
--arm
--armt
--ppc
--sparc
--ia64
--delta [1..32]
```
and
```
--dictsize 12-30 (expressed as 2^ds)
```

**Comment by pete4abw** [comment]
OR, is `liblrzip` even relevant anymore? I'm leaning to **NO**.

**Comment by pete4abw** [comment]
liblrzip and the demo programs have been abandoned. Right now, the files are archived and not used or compiled. They remain for historical purposes only.

**Comment by demhademha** [comment]
Where is the source code for the library?
I checked in `./lib`, but there is only the source code for `libzpaq`
I **may** be able to help with updating the library

**Reply by pete4abw** [reply]
Try the [src/libdemo directory](https://github.com/pete4abw/lrzip-next/tree/master/src/libdemo), or you can checkout an earlier commit before I nuked it. Like, e87a920. No one used it and it was just a waste of space. Also, I did not want to maintain it because lrzn got more complex.

**Reply by demhademha** [reply]
I suppose libarchive exists.

**Reply by pete4abw** [reply]
libarchive? Should be a standard distribution library. I would not waste effort on it. MAYBE when everything gets finalized...

Thank you for the offer. Using and Testing `lrzip-next` is the best anyone can offer right now! Along with additional suggestions.
